app/assets/javascripts/discourse/app/components/sidebar.hbs

```diff
@@ -4,9 +4,28 @@
   @class="sidebar-container"
   @scrollTop={{false}}
 >
-  <Sidebar::Sections
-    @currentUser={{this.currentUser}}
-    @collapsableSections={{true}}
-  />
+  {{#if this.showMainPanel}}
+    <Sidebar::Sections
+      @currentUser={{this.currentUser}}
+      @collapsableSections={{true}}
+      @panel={{this.currentPanel}}
+    />
+  {{else}}
+    <Sidebar::ApiPanels
```

Comment:
I think we need to think about what if there are multiple panels being registered. How do we determine which panel should be display since we can only display one panel each time.

Reply:
`this.currentPanel` decide what is displayed. Logic is here:

https://github.com/discourse/discourse/pull/22534/files#diff-af117902d76d9577ae269a34ee6c5a7f3df5d771c54ebd23300e997dfd3796f1R13